tests/creator/ServerRequestCreatorTest.php

```
@@ -1113,6 +1114,37 @@ public function testCreateFromGlobalsWithServerValues(): void
+
+        self::assertInstanceOf(
+            UploadedFileInterface::class,
+            $uploadedFile,
+            "Should create an 'UploadedFileInterface' for failed upload entries.",
+        );
+        self::assertSame(
+            UPLOAD_ERR_NO_FILE,
+            $uploadedFile->getError(),
+            "Should preserve the failed 'upload error' code during request creation.",
+        );
+    }
```

PHPMD (2.15.0)

[error] testCreateFromGlobalsWithUploadErrorDoesNotOpenEmptyTmpName accesses the super-global variable $_FILES. (undefined)

(Superglobals)